client/v2/indexer/lookupAccountByID.go

```diff
@@ -29,6 +34,14 @@ type LookupAccountByID struct {
 	p LookupAccountByIDParams
 }
 
+// Exclude exclude additional items such as asset holdings, application local data
+// stored for this account, asset parameters created by this account, and
+// application parameters created by this account.
+func (s *LookupAccountByID) Exclude(Exclude []string) *LookupAccountByID {
```

Comment:
why is this a list of strings?

Reply:
This code is generated. I didn't have a lot of choice...

Reply:
You can provide multiple Exclude values, i.e. `created-assets,created-apps`

Reply:
Do you mean 4 arguments for `Exclude()`?